tests/testsuite/bad_config.rs

```diff
@@ -442,7 +447,13 @@ Caused by:
   could not parse input as TOML
 
 Caused by:
-  expected a table key, found a newline at line 8 column 27
+  TOML parse error at line 8, column 27
+    |
+  8 |                 native = {
+    |                           ^
+  Unexpected `
+  `
```

Comment:
I think this should say "Unexpected newline" rather than backquoting a literal newline.

Reply:
On top of my above comments on `combine`, we do have a more specific issue for this: https://github.com/ordian/toml_edit/issues/259